app/services/ncr/work_order_value_normalizer.rb

```diff
@@ -41,6 +42,12 @@ def normalize_soc_code
       end
     end
 
+    def normalize_approving_official_email
+      if !work_order.approving_official_email.present? && work_order.approvers.any?
+        work_order.approving_official_email = work_order.approvers.first.try(:email_address)
```

Comment:
same reason it was originally needed in the `after_initialize` callback, I guess. All I know is lots of tests start failing w/o it.